packages/terra-brand-footer/src/BrandFooter.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './BrandFooter.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+ /**
+  * An array of navigation links with each element specifiying text, href and target keys with appropriate values.
+  */
+  links: PropTypes.array,
```

Comment:
If the links prop type is expecting an array of objects with `text`, `href`, and `target` keys, we should expand this prop type to be:
```
links: PropTypes.arrayOf(PropTypes.shape(
  /* description */
  text: TYPE,
  /* description */
  href: TYPE,
  /* description */
  target: TYPE,
)),
```

Reply:
@emilyrohrbough Discussed this with @bjankord and concluded that not all objects in the array will have target.

Reply:
Even if not all links need the `target` key, we can still define it via proptypes as long as we don't add `.isRequired` at the end of the proptype declaration.

Reply:
+1, @msaikaushik1992 let's add `PropTypes.arrayOf(PropTypes.shape` code Emily posted above. All of the props in the shape can be left optional.